Update outdated CLI edit releases information (#30241)
github/docs: content/repositories/releasing-projects-on-github/managing-releases-in-a-repository.md

{% cli %}

> 1. To edit a release, use the `gh release edit` subcommand. Replace `tag` with the tag representing the release you wish to edit. For example, to edit a given release's title:

Reviewer: ```suggestion
1. To edit a release, use the `gh release edit` subcommand. Replace `TAG` with the tag representing the release you wish to edit. For example, to edit a given release's title, use the following code:
```

Just a couple small tweaks for ease of localization, and to adhere to our style guide.

> 1. To edit a release, use the `gh release edit` subcommand. Replace `TAG` with the tag representing the release you wish to edit. For example, to edit a given release's title, use the following code:

Reviewer: ```suggestion
1. To edit a release, use the `gh release edit` subcommand. Replace `TAG` with the tag representing the release you wish to edit. For example, to edit the title for a release, use the following code, replacing `NEW-TITLE` with the updated title:
```

   ```shell

>    gh release edit <tag> -t "New title"

Reviewer: ```suggestion
   gh release edit TAG -t "New title"
```

One tiny change to align the formatting with our [style guide](https://docs.github.com/en/contributing/style-guide-and-content-model/style-guide#placeholders).

>    gh release edit TAG -t "New title"

Reviewer: ```suggestion
   gh release edit TAG -t "NEW-TITLE"
```

   ```

>    For more information about possible arguments, see [the {% data variables.product.prodname_cli %} manual](https://cli.github.com/manual/gh_release_edit)

Reviewer: ```suggestion
   For more information about possible arguments, see [the {% data variables.product.prodname_cli %} manual](https://cli.github.com/manual/gh_release_edit).
```

Small typo fix.
